.capstanignore

```diff
@@ -0,0 +1,9 @@
+# Exclude what we don't need in unikernel
```

Comment:
We have to wait for Capstan to support ignore, in particular since you are now changing it a bit so these patterns may not be valid anymore.